proxyprefix/contrib/djproxy.py

```diff
@@ -0,0 +1,15 @@
+class XForwardedPrefix(object):
+    """Add an X-Forwarded-Prefix header to the djproxy upstream request.
+
+    For enabling proxyprefix behavior during local development (where you may
+    not have apache/ngingx/whatever passing the header for you).
+
+    """
+
+    def process_request(self, proxy, request, **kwargs):
+        for prefix, url in proxy.reverse_urls:
+            if request.path.startswith(prefix):
```

Comment:
Sorry, I meant that in my original pull request to djproxy I was checking both proxy_url and request.path, but I didn't think I needed to check proxy_url after all, so I left that out here, but wanted to check with you in case I was missing something. But it looks like you answered that in your other comment.